### PR TITLE
MCLOUD-1528: Change default value for max execution time of static content deploy

### PR DESCRIPTION
**Pull request by dhorytskyi**
## Purpose of this pull request

Change default value for max execution time of static content deploy

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/env/variables-deploy.html#scd_max_execution_time
- https://devdocs.magento.com/cloud/env/variables-global.html#scd_max_execution_time
- https://devdocs.magento.com/cloud/env/variables-build.html#scd_max_execution_time

## Links to Magento source code

https://github.com/magento/magento2ce/blob/2.3-develop/app/code/Magento/Deploy/Process/Queue.php#L32

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by meker12** [comment]
@dhorytskyi  I see that the source code change is on the 2.3-develop branch.  When will this change be released to production, for what versions?

**Comment by meker12** [comment]
OK -- changed the base branch for this PR to the 2.4.1 develop branch.  @dhorytskyi  To help current customers, I added a note to the current docs with info about increasing the value.  Please review PR #7624 .

Thanks for proactively notifying us about the doc change :-)

File: src/_data/codebase/v2_3/commerce/bin-magento.json [hunk 1]
```diff
@@ -10718,7 +10718,7 @@
                   "is_value_required" : false,
                   "description" : "The maximum expected execution time of deployment static process (in seconds).",
                   "accept_value" : true,
-                  "default" : 400,
+                  "default" : 900,
```
Comment (meker12) [inline src/_data/codebase/v2_3/commerce/bin-magento.json:10721]: Need to revert the changes to the .json files:

The content in thes files is auto-generated from the Magento codebase.


Any manual changes will be overwritten the next time we release. Additionally, manual changes might not accurately reflect what is in the code base for a release, so the docs will not be accurate.

Comment (meker12) [inline src/_data/codebase/v2_3/commerce/bin-magento.json:10721]:
```suggestion
                  "default" : 400,
```

File: src/_data/codebase/v2_3/open-source/bin-magento.json [hunk 2]
```diff
@@ -10476,7 +10476,7 @@
                "max-execution-time" : {
                   "accept_value" : true,
                   "shortcut" : "",
-                  "default" : 400,
+                  "default" : 900,
```
Comment (meker12) [inline src/_data/codebase/v2_3/open-source/bin-magento.json:10479]:
```suggestion
                  "default" : 400,
```

File: src/_data/codebase/v2_3/commerce/bin-magento.json (hunk 1 above)
Comment (dhorytskyi) [reply src/_data/codebase/v2_3/commerce/bin-magento.json]: Reverted

**Comment by dhorytskyi** [comment]
@meker12 It will be changed starting from 2.3.6 and 2.4.1.
Also, I reverted changes from json files.

**Comment by dobooth** [comment]
running tests
